modules/mono/glue/cs_files/Color.cs

```diff
@@ -45,8 +45,8 @@ public float h
         {
             get
             {
-                float max = (float) Mathf.Max(r, (float) Mathf.Max(g, b));
-                float min = (float) Mathf.Min(r, (float) Mathf.Min(g, b));
```

Comment:
Thanks! I'll also check it again to see if I've missed similar cases elsewhere.

By the way, would it be better if we use `Math` instead of `Mathf` where we know for certain that the values are `float`s?

The reason is, it's easy to miss such cases because the compiler won't complain unless you double check with `REAL_T_IS_DOUBLE` while leaving the casts will make the code inspector issue a false positive unless we use a vendor specific directives in a comment.

I assume that the reason why `min`, `max` weren't declared as `real_t` is that it'd be an overkill to use for RGB colors. So, in that case, would it make sense to use `System.Math` instead of `Godot.Mathf`?

Reply:
`System.Math` works with `double` exclusively. Godot's `Mathf` can work with either. It doesn't make any sense to use `System.Math` as we'd have lower (or equal) performance and less control over the code. All you'd accomplish is causing code-checking tools to shut up about it.

Hopefully we will add compiler checks for double-precision later on.

Reply:
Ah, I see. I assumed it'd have `float` version override without actually checking it, sorry!

I'll ask if it'd be ok to use ReSharper directives to suppress false positives like these then. Thanks for the clarification.

Reply:
@mysticfall I was wrong, in this particular case, it does have overloads. A lot of `System.Math` works with only `double` though, so just check each time :)

Reply:
Ok, got it. Thanks! :)